src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/ansys_sphinx_theme.css

```
@@ -1161,3 +1161,9 @@ a:visited:hover {
 .bd-sidebar-primary {
   padding: 0.5rem;
 }
+
+/* Auto-api prename */
```

Let's provide a more descriptive comment on what this is targeting at:

```suggestion
/* Do not display object domain namespace when documenting objects.

   Example: class ansys.module.foo.Foo.method -> method

  */
```